src/suggest-edit.js

```diff
@@ -129,14 +131,20 @@ function renderForm(schema, company = undefined) {
 
 document.getElementById('submit-suggest-form').onclick = () => {
     let data = bf.getData();
+    /* eslint-disable-next-line no-unused-vars */
+    const preact = require('preact');
     if (!data) {
-        /* eslint-disable no-unused-vars */
-        let preact = require('preact');
-        /* eslint-enable no-unused-vars */
         flash(<FlashMessage type="warning">{t('no-input', 'suggest')}</FlashMessage>);
         return;
+    } else if (!data.name && !data.web) {
+        flash(<FlashMessage type="warning">{t('name-or-web-missing', 'suggest')}</FlashMessage>);
+        return;
     }
 
+    document.querySelectorAll('.invalid').forEach((el) => {
```

Comment:
*Most likely* this will not be a problem but I am little worried about removing **all** `.invalid` classes. Couldn't we make the selector more specific just in case?

```suggestion
    document.querySelectorAll('.brutusin-form .invalid').forEach((el) => {
```